app/src/main/res/navigation/nav_graph.xml

```diff
@@ -40,17 +40,47 @@
         android:id="@+id/transactions_fragment"
         android:name="com.kwartracker.android.transactions.ui.main.TransactionsFragment"
         android:label="Transaction"
-        tools:layout="@layout/fragment_transaction" />
+        tools:layout="@layout/fragment_transaction" >
+        <action
+            android:id="@+id/action_transactions_fragment_to_transaction_add_fragment"
+            app:destination="@id/transaction_add_fragment" />
+    </fragment>
 
     <fragment
         android:id="@+id/transaction_add_fragment"
         android:name="com.kwartracker.android.transactions.ui.add.TransactionAddFragment"
         android:label="Transaction Add"
-        tools:layout="@layout/fragment_transaction_add" />
+        tools:layout="@layout/fragment_transaction_add" >
+        <action
+            android:id="@+id/action_transaction_add_fragment_to_transaction_add_1_fragment"
+            app:destination="@id/transaction_add_1_fragment" />
+    </fragment>
 
     <fragment
         android:id="@+id/transaction_add_1_fragment"
         android:name="com.kwartracker.android.transactions.ui.add.TransactionAddFragmentS1"
         android:label="Transaction Add"
         tools:layout="@layout/fragment_transaction_add_1" />
+
+    <fragment
+        android:id="@+id/profile_fragment"
+        android:name="com.kwartracker.android.profile.ui.MyProfileFragment"
+        android:label="Transaction"
+        tools:layout="@layout/fragment_my_profile" />
+
+    <fragment
+        android:id="@+id/dashoardFragment"
+        android:name="com.kwartracker.android.dashboard.ui.DashboardFragment"
+        android:label="Login"
+        tools:layout="@layout/fragment_dashboard" >
+        <action
+            android:id="@+id/action_dashoardFragment_to_walletsFragment"
+            app:destination="@id/walletsFragment" />
```

Comment:
pls use snake case

Reply:
fix in 49df7d5e62794cf0a6698424f963f76f5828af96